app/cdap/components/Metadata/index.tsx

```diff
@@ -49,23 +49,20 @@ const Metadata: React.FC = () => {
         <Route
           exact
           path={basepath}
-          render={() => checkFeatureFlag(<MetadataHome />, `/cdap/ns/${namespace}/metadata`)}
+          render={() => checkFeatureFlag(<MetadataHome />, `/metadata/ns/${namespace}`)}
```

Comment:
These are fallback values when the feature flag `isMetadataInReact` is turned off. Should navigate to angular metadata pages.